README.md

```diff
@@ -42,12 +42,37 @@ More information about Black, you find at
 The following software is required to run PTF:
 
  * Python 2.7 or 3.x
- * Scapy 2.4.4
+ * Scapy 2.4.5
+ * isort 1.16.0
  * pypcap (optional - VLAN tests will fail without this)
  * tcpdump (optional - Scapy will complain if it's missing)
 
 Root/sudo privilege is required on the host, in order to run `ptf`.
 
+To install minimal requirements execute:
+```text
+pip install -r requirements.txt
+```
+
+To enable VLAN tests, you need to install `pycap`:
+```text
+pip install PyCap
+```
```

Comment:
@antoninbas @NexSabre 
I think I missed this part, why do we need PyCap?
According to the PyCap readme(https://pypi.org/project/PyCap/)

```
PyCap is a python module exposing the REDCap API through some helpful abstractions.
```

Is REDCap related to the packet test? And can you point me out where is the VLAN test you mentioned here?

Reply:
It's a typo, looks like it should be ` pypcap`, not `pycap`.